### PR TITLE
Update genesis.tmpl.json and accounts.csv for final genesis (prelaunch & mainnet)

### PR DESCRIPTION
**Pull request by clevinson**
Two main categories of updates happening here:
- Updating `regen-prelaunch-1` genesis.tmpl.json to match the recently updated genesis params in #110 #103
- Updating accounts.csv with necessary updates from the initial candidate chain account addresses & balances:
  - two missing token allocations were added
  - Tokens in RND's Community Bootstrapping Fund decreased by 7207 REGEN, as this address was holding the extra funds for the missing token allocations
  - one token holder needed their addresses updated
  - two token holders needed their allocation split between two addresses
  
After the above changes were made to the `accounts.csv` and `genesis.tmpl.json`, I reran the genesis script for both `regen-1` and `regen-prelaunch-1` directories.

I've set a new genesis time for a second candidate chain to occur at: `2021-04-09T16:00:00Z`.

@anilCSE, we should merge this in and boot up the new chain tomorrow against a new regen-ledger binary (with the recent tendermint security fix from SDK v0.42.4). This should be `v1.0.0-rc3`.

### CODE REVIEW

**Review by aaronc — COMMENTED** [review]
Why overwrite regen-prelaunch-1? Why not create a separate regen-prelaunch-2 folder?

**Comment by clevinson** [comment]
Ok, i think that might be cleaner. Will go ahead and update.

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,12 @@
 all: prelaunch
```
Comment (aaronc) [inline Makefile]:
```suggestion
```

**Review by aaronc — COMMENTED** [review]
Why both genesis.json and genesis-prelaunch.json in regen-1?

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,11 @@
-all: prelaunch
 
-.PHONY: prelaunch
+.PHONY: prelaunch prelaunch-2 regen-1
```
Comment (aaronc) [inline Makefile]:
```suggestion
.PHONY: prelaunch-2 regen-1
```

File: Makefile [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
 
-prelaunch:
-	go run . build-genesis regen-prelaunch-1 --errors-as-warnings
\ No newline at end of file
+prelaunch-1:
+	go run . build-genesis regen-prelaunch-1 --errors-as-warnings
```
Comment (aaronc) [inline Makefile]:
```suggestion
```

**Comment by clevinson** [comment]
i decided to drop the `regen-1` make target entirely, as it creates the genesis.json file. We should add it back in once we have all the gentx PRs merged, and this make task can be used to create the final genesis for mainnet.

**Review by aaronc — APPROVED** [review]

**Pull request merged**